qtgui/resources/PixelColors.qml

```diff
@@ -0,0 +1,59 @@
+import QtQuick 2.12
+import QtQuick.Controls 2.14
+import QtQuick.Dialogs 1.3
+import QtQuick.Layouts 1.14
+
+Frame {
+	property var color1: color("black")
+	property var color2: color("white")
+    property var colorToChange: undefined
+    property var labelToChange: undefined
+	
+    GridLayout {
+        columns: 2
+        Rectangle {
+            id: lbcol1

```

Comment:
this is really boiler-platey. but I trust that'll be taken care of when extending the color list (make colors addable etc.)